Classes/Service/Tika/ServerService.php

```diff
@@ -198,7 +217,17 @@ public function isAvailable()
      *
      * @return string Tika server URL
      */
-    public function getTikaServerUrl()
+    public function getTikaServerUrl(): string
+    {
+        return (string)$this->tikaUrl;
+    }
+
+    /**
+     * Constructs the Tika server Uri.
+     *
+     * @return Uri Tika server Uri
+     */
+    public function getTikaServerUri(): Uri
```

Comment:
With the new usage of the Uri object the naming of the properties and methods is kind of confusing,  e.g. `tikaUrl` contains the Uri and getTikaServerUri  returns `tikaUrl`.

Since we have adjusted the behavior of some internal functions anyway, we could also rename the property to `tikaUri` to clear up the confusion a bit or alternatively do without the getter for Uri. What do you think?

Reply:
Property name changed to tikaUri.